### PR TITLE
orca pytorch estimator predict

### PR DESCRIPTION
**Pull request by cyita**

### CODE REVIEW

File: pyzoo/zoo/orca/data/utils.py [hunk 1]
```diff
@@ -194,8 +194,12 @@ def to_sample(data):
     from bigdl.util.common import Sample
     data = check_type_and_convert(data, allow_list=True, allow_tuple=False)
     features = data["x"]
-    labels = data["y"]
     length = features[0].shape[0]
+    has_label = "y" in data
+    if has_label:
+        labels = data["y"]
+    else:
+        labels = np.zeros([1, length])
```
Comment (qiuxin2012) [inline pyzoo/zoo/orca/data/utils.py:202]: a invalid value like -1 is better.

File: pyzoo/zoo/orca/data/utils.py [hunk 2]
```diff
@@ -194,8 +194,12 @@ def to_sample(data):
     from bigdl.util.common import Sample
     data = check_type_and_convert(data, allow_list=True, allow_tuple=False)
     features = data["x"]
-    labels = data["y"]
     length = features[0].shape[0]
+    has_label = "y" in data
+    if has_label:
```
Comment (qiuxin2012) [inline pyzoo/zoo/orca/data/utils.py:199]: why not `if "y" in data`?

**Comment by cyita** [comment]
Jenkins passed: http://10.239.47.210:18888/job/ZOO-PR-Validation/4754/

File: pyzoo/zoo/orca/learn/pytorch/estimator.py [hunk 3]
```diff
@@ -251,8 +251,16 @@ def fit(self, data, epochs=1, batch_size=32, validation_data=None, validation_me
...
 
-    def predict(self, data, **kwargs):
-        pass
+    def predict(self, data, batch_size=4):
+        from zoo.orca.learn.utils import convert_predict_to_xshard
+        if isinstance(data, SparkXShards):
+            from zoo.orca.data.utils import to_sample
+            data_rdd = data.rdd.flatMap(to_sample)
+        else:
+            raise ValueError("Data should be SparkXShards, each element needs to be {'x': a "
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/pytorch/estimator.py:260]: Data should be XShards

**Pull request merged**